src/drivers/indexeddb.js

```diff
@@ -994,42 +1019,49 @@ function dropInstance(options, callback) {
                     return;
                 }
 
-                var newVersion = db.version + 1;
+                const newVersion = db.version + 1;
 
                 _deferReadiness(options);
 
-                var dbContext = dbContexts[options.name];
-                var forages = dbContext.forages;
+                const dbContext = dbContexts[options.name];
+                const forages = dbContext.forages;
 
-                for (var i = 0; i < forages.length; i++) {
-                    var forage = forages[i];
-                    if (forage._dbInfo.db) {
-                        forage._dbInfo.db.close();
-                        forage._dbInfo.db = null;
-                        forage._dbInfo.version = newVersion;
-                    }
+                db.close();
+                for (let i = 0; i < forages.length; i++) {
+                    const forage = forages[i];
+                    forage._dbInfo.db = null;
+                    forage._dbInfo.version = newVersion;
                 }
 
-                var dropObjectPromise = new Promise(function(resolve, reject) {
-                    var req = idb.open(options.name, newVersion);
+                const dropObjectPromise = new Promise(function(
+                    resolve,
+                    reject
+                ) {
+                    const req = idb.open(options.name, newVersion);
 
-                    req.onerror = reject;
+                    req.onerror = e => {
```

Comment:
As mentioned too, I'd like to see us wrap things like this `e` in parens and use no single-letter variables, so it'd be cool if this could be `req.onerror = (err) => {`